www/pfSense-pkg-Lightsquid/files/usr/local/pkg/lightsquid.inc

```diff
@@ -328,12 +335,16 @@ ssl.honor-cipher-order	= "enable"
 ssl.cipher-list		= "AES128+EECDH:AES256+EECDH:AES128+EDH:AES256+EDH:AES128-SHA:AES256-SHA:!aNULL:!eNULL:!DSS"
 EOF;
 
-		// Configure IPv4 and IPv6 sockets
+		/*
+		 * IPv4 HTTPS bind must NOT be a socket; otherwise the retarded
+		 * lighttpd thing yet again binds to 80/HTTP no matter what.
+		 * You enable SSL-only IPv4 by not configuring bind for IPv4 at all.
+		 * IPv6 SSL must be configured as a socket
+		 */
 		$lighttpd_ls_conf .= <<<EOF
 
```

Comment:
Yes, server.port defaults to port 80 and lighttpd will bind to \*:80 or whatever server.port is set to.  If you do not want the default behavior of binding to port 80, then you need to specify server.port and, optionally, server.bind.  You can also optionally specify a repeat of the implicit bind to "\*:$port" in a $SERVER["socket"] block.

You do not need to repeat $lighttpd_ls_conf_sslcommon inside this block if it is already present in the global scope.

```
server.port		= "{$lightsquid_config['lighttpd_ls_port']}"
# \$SERVER["socket"]	== "0.0.0.0:{$lightsquid_config['lighttpd_ls_port']}" { }
\$SERVER["socket"]	== "[::]:{$lightsquid_config['lighttpd_ls_port']}" { }
{$lighttpd_ls_conf_sslcommon}
```